addons/website/static/src/scss/website.backend.scss

```diff
@@ -373,3 +373,71 @@
         }
     }
 }
+
+.o_view_form_theme_preview_controller {
+    .o_control_panel > div:first-of-type {
+        display: none;
+    }
+    div.o_form_nosheet {
+        padding: 0px;
+        height:100%;
+        width:100%;
+    }
+
+    .is_mobile {
+        @include media-breakpoint-up(md) {
+            iframe {
+                // mobile frame is rounded
+                border-radius: 15px;
+                height: 735px;
+            }
+            .img_mobile {
+                pointer-events: none;
+                display: block !important;
+                position: absolute;
+                top: 16px;
+                left: calc(50% - 200px)
+            }
+            .o_field_iframe_preview {
+                margin: auto !important;
+                padding: 53px 11px 58px 28px;
+                width: 416px;
+            }
+        }
+    }
+
+}
+.o_view_kanban_theme_preview_controller {
+    .o_searchview_more, .o_control_panel > div:nth-child(2) {
+        display: none;
+    }
+}
+
+.theme_install_loader {
+  display: inline-block;
+  position: relative;
+  width: 64px;
+  height: 64px;
+}
+.theme_install_loader:after {
+  content: " ";
+  display: block;
+  border-radius: 50%;
+  margin: 6px;
+  border: 26px solid $o-brand-primary;
+  border-color: $o-brand-primary transparent $o-brand-primary transparent;
+  animation: theme_install_loader 1.2s infinite;
+}
+@keyframes theme_install_loader {
+  0% {
+    transform: rotate(0);
+    animation-timing-function: cubic-bezier(0.55, 0.055, 0.675, 0.19);
+  }
+  50% {
+    transform: rotate(900deg);
+    animation-timing-function: cubic-bezier(0.215, 0.61, 0.355, 1);
+  }
+  100% {
+    transform: rotate(1800deg);
+  }
+}
```

Comment:
- Do important rules really need to be important? That's strange since those are your classes?
- Note: I try to respect an order for css rules (see first rule in https://github.com/odoo/odoo/wiki/SCSS-coding-guidelines) but that would be more important for website apps / themes :wink: 
- Review indentation and spaces after `:` ;)